### ISSUE TITLE
suggestion: wifi spammer 

### ISSUE THREAD

**Issue by Haxin-hipopcorn** [cSploit/android#344]
imagine, if you can make huge fake clients connecting to the wifi, the ITs guys will be crazy!!. i think it's an awesome feature to implement.

**Comment by Rubenoo** [comment]
Or like redirect everyone on the wifi to a custom page that you can make

**Comment by domenicoblanco** [comment]
You can already do this with the MITM when you attack the subnet

**Comment by domenicoblanco** [comment]
@Rubenoo You can already do this with the MITM when you attack the subnet

**Comment by Rubenoo** [comment]
Yeah but i meant that you can upload a custom webpage or site like in social engineering and get the results back you know

**Comment by Haxin-hipopcorn** [comment]
@Rubenoo you can already do that

**Comment by tux-mind** [comment]
@Haxin-hipopcorn I didn't understand what is the feature request.
please explain in detail and thanks for your time :wink:

**Comment by DeveloppSoft** [comment]
Maybe it was "wifi jammer"...
The goal is to disconnect all the client of an AP

**Comment by Haxin-hipopcorn** [comment]
@tux-mind fake  clients in the app, for example you run csploit and show a lot of fake targets, then you will make the owners of the network being confused

**Comment by tux-mind** [comment]
on a static ip LAN you'll get collisions with existing addresses ( admin get stats only if a sniffer it's installed on the router ).
on a DHCP LAN is enough to start many DHCP requests until the DHCP server finishes up it's pool.
this will also prevent other clients to connect to the network.

we can make it, but I really want to ask who else will find it useful.

**Comment by DeveloppSoft** [comment]
I think it is not, sorry.

**Comment by Haxin-hipopcorn** [comment]
@tux-mind it will be usefull to disguise, think with me, if they want to find you, spamming a lot of clients, they will cannot find you!!, or if they want to filter some macs... will be difficulty, or just to someone tries to search for exploits, or do an nmap, it will be sooo sloowww, so we have a lot of utilities.

**Comment by 0MazaHacka0** [comment]
It will be useful

**Comment by Rubenoo** [comment]
It would be nice if you could crash the router for a while :)

**Comment by newbie442** [comment]
ISTM the original post is very interesting - and likely a different tool than csploit.

IIUC, the suggested tool would create multiple connections with the WAP, using multiple spoofed MAC addresses, and each would effect traffic with the WAP - perhaps additionally signing in and effecting traffic with the WAN via the WAP.

This might do a couple of things:

1. Hide the csploit user from a whitehat in a crowded room.
2. Hide the csploit user from an attacker within the crowd.
 
(I actually doubt that 1. would succeed, but suspect that it could confuse would be attackers - "which target to attack?" This seems a very complex app)
